agile-development/team-agreements/team-manifesto/readme.md

```diff
@@ -0,0 +1,78 @@
+# Team Manifesto
+
+## Introduction
+
+CSE teams work with a new development team in each customer engagement which requires a phase of introduction & knowledge transfer before starting an engagement.
+
+Completion of this phase of ice-breakers and discussions around the standards takes time however also required to start increasing the learning curve of the new team.
+
+A team manifesto is a soft contract among team members which summarizes the basic principles and values of the team and aiming to provide a consensus about
+expectations from each team member in order to deliver high quality output at the end of each engagement.
+
+It aims to reduce the time on setting the right expectations and providing a consensus among team members as well as clarifying the question of "How does the new team develop the software?"
+
+Another main goal of the manifesto is providing a discussion environment during "manifesto building session" and detect the gaps/differences in certain topics for each member during these discussions.
+
+It also serves in the same way when a new team member joins to the team. It gives a short summary of standards and if the new team member asks any question about any topic, this is a good thing to understand the improvement area and take the required actions.
+
+## How to Build a Team Manifesto
+
+It can be said that the best time to start building it is at the very early phase of the engagement when teams meet with each other for swarming or during the preparation phase.
+
+It is recommended to keep team manifesto as simple as possible, so preferably, one-page simple document which **doesn't include any references or links** is a nice format for it.
+If there is a need for providing knowledge on certain topics, the way to do is delivering brown-bag sessions, technical katas, practices documentations and others later on.
+
+A few important points about team manifesto;
+
+- Team manifesto is built by the **team itself**
+
+- It can include any technical or behavioural agility topics as soon as team find it relevant,
+
+- It aims to have a common understanding about the desired expertise, practices and/or behaviours within the team
+
+- Based on the needs of the team and retrospective results, it can be modified during the engagement.
+
+In Microsoft, we have certain standards to build high quality and well-crafted software as well as to give comfortable/transparent environment to each team member for helping her/him to reach her/his highest potential.
```

Comment:
maybe simpler just to use "them" and "their" over he/she pronouns.

Reply:
fixed commit b07b8657c0ea24e73211d9bfc93994abe9cf3545